### PR TITLE
Cut 0.28.0-rc.0

### PR DESCRIPTION
**Pull request by yeya24**
Signed-off-by: Ben Ye <ben.ye@bytedance.com>

* [ ] I added CHANGELOG entry for this change.
* [ ] Change is not relevant to the end user.

## Changes

Cut 0.28.0-rc.0. Please help review it @thanos-io/thanos-maintainers.

### CODE REVIEW

**Review by bwplotka — APPROVED** [review]
💪🏽 Thanks!

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,40 +9,46 @@ NOTE: As semantic versioning states all 0.y.z releases can contain breaking chan
...
-- [#5475](https://github.com/thanos-io/thanos/pull/5475) Compact/Store: Added `--block-files-concurrency` allowing to configure number of go routines for download/upload block files during compaction.
-- [#5470](https://github.com/thanos-io/thanos/pull/5470) Receive: Implement exposing TSDB stats for all tenants
-- [#5493](https://github.com/thanos-io/thanos/pull/5493) Compact: Added `--compact.blocks-fetch-concurrency` allowing to configure number of go routines for download blocks during compactions.
-- [#5480](https://github.com/thanos-io/thanos/pull/5480) Query: Expose endpoint info timeout as a hidden flag.
-- [#5527](https://github.com/thanos-io/thanos/pull/5527) Receive: Add per request limits for remote write.
-- [#5520](https://github.com/thanos-io/thanos/pull/5520) Receive: Meta-monitoring based active series limiting.
+- [#5472](https://github.com/thanos-io/thanos/pull/5472) Receive: Add new tenant metrics to example dashboard.
+- [#5475](https://github.com/thanos-io/thanos/pull/5475) Compact/Store: Added `--block-files-concurrency` allowing to configure number of go routines for downloading and uploading block files during compaction.
```
Comment (fpetkovski) [inline CHANGELOG.md:29]:
```suggestion
- [#5475](https://github.com/thanos-io/thanos/pull/5475) Compact/Store: Added `--block-files-concurrency` allowing to configure number of goroutines for downloading and uploading block files during compaction.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -9,40 +9,46 @@ NOTE: As semantic versioning states all 0.y.z releases can contain breaking chan
...
+- [#5470](https://github.com/thanos-io/thanos/pull/5470) Receive: Implement exposing TSDB stats for all tenants.
```
Comment (fpetkovski) [inline CHANGELOG.md:30]:
```suggestion
- [#5470](https://github.com/thanos-io/thanos/pull/5470) Receive: Expose TSDB stats as metrics for all tenants.
```

**Review by fpetkovski — APPROVED** [review]
Small nits, otherwise looks good. So many features 🚀

File: CHANGELOG.md (hunk 2 above)
Comment (yeya24) [reply CHANGELOG.md:30]: Fixed.

File: CHANGELOG.md (hunk 1 above)
Comment (yeya24) [reply CHANGELOG.md:29]: Fixed.

**Review by kakkoyun — APPROVED** [review]

File: CHANGELOG.md [hunk 3]
```diff
@@ -9,40 +9,46 @@ NOTE: As semantic versioning states all 0.y.z releases can contain breaking chan
...
+- [#5480](https://github.com/thanos-io/thanos/pull/5480) Query: Expose endpoint info timeout as a hidden flag `--endpoint.info-timeout`.
+- [#5527](https://github.com/thanos-io/thanos/pull/5527) Receive: Add per request limits for remote write. Added four new flags `--receive.write-request-limits.max-size-bytes`, `--receive.write-request-limits.max-series`, `--receive.write-request-limits.max-samples` and `--receive.write-request-limits.max-concurrency` for limiting requests max body size, max amount of series, max amount of samples and max amount of concurrent requests.
+- [#5520](https://github.com/thanos-io/thanos/pull/5520) Receive: Meta-monitoring based active series limiting. This mode is only available if Receiver is in Router or RouterIngestor mode, and config is provided. Added four new flags `receive.tenant-limits.max-head-series` for the max active series for the tenant, `receive.tenant-limits.meta-monitoring-url` for the Meta-monitoring URL, `receive.tenant-limits.meta-monitoring-query` for specifying the PromQL query to execute and `receive.tenant-limits.meta-monitoring-client` for specifying HTTP client configs.
```
Comment (matej-g) [inline CHANGELOG.md:34]: Should we mention that these features are exeprimental right now / the flags are hidden?

**Review by matej-g — APPROVED** [review]
Looking good :+1: One optional suggestion.

File: CHANGELOG.md (hunk 3 above)
Comment (yeya24) [reply CHANGELOG.md]: Make sense. Updated!
But it is still not very clear what features are hidden now in Thanos. In the Query part, we have feature flags to align with Prometheus. Here we are using hidden flags. Should we choose one way to go in the future?

**Comment by yeya24** [comment]
Let's merge on green :)

File: CHANGELOG.md (hunk 3 above)
Comment (matej-g) [reply CHANGELOG.md]: Definitely :+1: I didn't realize we are using two different ways, I'll put in an issue to resolve this.

**Comment by yeya24** [comment]
Updated the branch to include the lastest 2 prs on main https://github.com/thanos-io/thanos/pull/5604 and https://github.com/thanos-io/thanos/pull/5607. Please take another look!

**Pull request merged**